PhysicsTools/HepMCCandAlgos/test/HZZ4muAnalyzer.h

```diff
@@ -1,15 +1,15 @@
 #ifndef HZZ4muAnalyzer_H
 #define HZZ4muAnalyzer_H
 
-#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 
 #include "SimDataFormats/GeneratorProducts/interface/HepMCProduct.h"
 
 // forward declarations
 class TFile;
 class TH1D;
 
-class HZZ4muAnalyzer : public edm::EDAnalyzer {
+class HZZ4muAnalyzer : public edm::one::EDAnalyzer<> {
```

Comment:
there is also GeneratorInterface/Pythia6Interface/test/HZZ4muAnalyzer.cc

while this one becomes a plugin via
```C++
typedef HZZ4muAnalyzer HZZ4muTestAnalyzer;
DEFINE_FWK_MODULE(HZZ4muTestAnalyzer);
```

wouldn't this still potentially lead to symbol conflicts; perhaps more certain if someone attempts to load both analyzers.

the GeneratorInterface one goes as 
```C++
typedef HZZ4muAnalyzer HZZ4muExampleAnalyzer;
DEFINE_FWK_MODULE(HZZ4muExampleAnalyzer);
```

Looking in history, the only use case of this analyzer in PhysicsTools/HepMCCandAlgos/test/GenH190ZZ4muHepMCAnalysis.py was removed in #10030

OTOH, the other one in GeneratorInterface did not even go through the `consumes` migration.